x-pack/plugins/cases/public/components/case_view/components/case_view_alerts.tsx

```diff
@@ -22,34 +24,49 @@ interface CaseViewAlertsProps {
 export const CaseViewAlerts = ({ caseData }: CaseViewAlertsProps) => {
   const { triggersActionsUi } = useKibana().services;
 
+  const alertIds = getManualAlertIds(caseData.comments);
   const alertIdsQuery = useMemo(
     () => ({
       ids: {
-        values: getManualAlertIds(caseData.comments),
+        values: alertIds,
       },
     }),
-    [caseData.comments]
+    [alertIds]
   );
 
-  const alertRegistrationContexts = useMemo(
-    () => getRegistrationContextFromAlerts(caseData.comments),
-    [caseData.comments]
+  const { isLoading: isLoadingAlertFeatureIds, data: alertData } = useGetFeatureIds(
+    alertIds,
+    caseData.owner !== SECURITY_SOLUTION_OWNER
   );
 
-  const { isLoading: isLoadingAlertFeatureIds, data: alertFeatureIds } =
-    useGetFeatureIds(alertRegistrationContexts);
-
   const configId =
-    caseData.owner === SECURITY_SOLUTION_OWNER ? `${caseData.owner}-case` : caseData.owner;
+    caseData.owner === SECURITY_SOLUTION_OWNER
+      ? `${caseData.owner}-case`
+      : !isLoadingAlertFeatureIds
```

Comment:
I will remove that when umbo PR is merged and we can have a generic config id working